jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/BytecodeAnnotator.java

```diff
@@ -280,7 +280,7 @@ public static void annotateBytecodeInJar(
     // Reference: https://bugs.openjdk.java.net/browse/JDK-8215788
     // Note: we can't just put the code below inside stream().forach(), because it can throw
     // IOException.
-    for (JarEntry jarEntry : inputJar.stream().collect(ImmutableList.toImmutableList())) {
+    for (JarEntry jarEntry : (Iterable<JarEntry>) inputJar.stream()::iterator) {
```

Comment:
There's one more that can be changed:
https://github.com/uber/NullAway/blob/7f9aacb6b6cf54e1b6765cfe87cdba20dcb58985/jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/AnnotationChecker.java#L90